status-server/src/public/js/client.js

```diff
@@ -1,10 +1,10 @@
 var intervalID;
 var statusItems = {
-  '/status0.json': {},  // statusData for each person
-  '/status1.json': {},  // statusData for each person
-  '/status2.json': {},  // statusData for each person
-  '/status3.json': {},  // statusData for each person
-  '/status4.json': {}  // statusData for each person
+  '/user/0/status': {},  // statusData for each person
```

Comment:
Sorry you have to rebase and resolve conflicts one more time - we just added user/5 and user/6